LockExample/FileService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using LockExample.Utils;
+[assembly: InternalsVisibleTo("LockExample.UnitTests")]
+namespace LockExample
+{
+    internal sealed class FileService
+    {
+        //private const string FolderName = "LockStatementApp";
+        //private const string Destination = "Destination";
+        private readonly string _sourcePath;
+        public FileService(string? sourcePath = null)
+        {
+            _sourcePath = sourcePath ?? Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+        }
+        public async Task StartFileCopy(string destinationFilePath, string destinationFolderName, string sourceFolderName)
```

Comment:
here likely having 2 parameters should be fine
- source directory path
- final directory path

this way we can copy any directory to anywhere